WebRoot/WEB-INF/tags/img.tag

```diff
@@ -40,9 +40,9 @@
         <c:otherwise> src="${info.src}" </c:otherwise>
     </c:choose>
     <c:choose>
-        <c:when test="${not empty title}"> title='${title}' </c:when>
-        <c:when test="${not empty alt}"> title='${alt}' </c:when>
+        <c:when test="${not empty title}"> title='${fn:escapeXml(title)}' </c:when>
+        <c:when test="${not empty alt}"> title='${fn:escapeXml(alt)}' </c:when>
     </c:choose>
     <c:if test="${not empty alt}"> alt="${fn:escapeXml(alt)}" </c:if>
-    <c:if test="${not empty clazz}"> class='${clazz}' </c:if>
-    <c:forEach items="${dynattrs}" var="a"> ${a.key}="${a.value}" </c:forEach> >
\ No newline at end of file
+    <c:if test="${not empty clazz}"> class='${fn:escapeXml(clazz)}' </c:if>
+    <c:forEach items="${dynattrs}" var="a"> ${a.key}="${fn:escapeXml(a.value)}" </c:forEach> >
```

Comment:
aren't these dynattrs coming from our server? how can we exploit this?

Reply:
Fixed